### DISCUSSION TITLE
Passing down events to child GestureDetector

### DISCUSSION THREAD

**Discussion by MrSquaare** [software-mansion/react-native-gesture-handler#2500] [Q&A]
Hello :wave: 

I have a `GestureDetector` which tracks pan gesture. It wraps a child which itself has a `GestureDetector` which tracks pan gesture, but I don't have control on it.
Can I let my parent `GestureDetector` pass down pan events to the children?

**Comment by j-piasecki** [comment]
Sorry, I don't think I understand the problem. Could you prepare a snack of post a snipped of code that shows the behavior you're getting and explain what you want it to look like?
